backend/data/processors/manualFixes/COMMFixes.py

```diff
@@ -0,0 +1,333 @@
+        "processed": "COMM6000 CA:Essentials || COMM1999 First Year Portfolio && 48UOC && 65WAM && be in good academic standing. who have COMM2233 || CDEV3000 (formerly DIPP1510) must not enrol into this course"
+    """
+    COURSES["COMM1150"]["exclusions"]["COMM2233"] = 1
+    COURSES["COMM1150"]["exclusions"]["CDEV3000"] = 1
+    COURSES["COMM1150"]["exclusions"]["DIPP1510"] = 1
+
+    return "(COMM6000 || COMM1999) && 65WAM && 48 UOC"
+
+def COMM_3020(conditions):
+    """
+        "original": "WAM 65+, minimum 72 UOC, room in degree for this course, good academic standing, completed COMM6000 Career Accelerator: Essentials or COMM1999 First Year Portfolio.<br/>This course is by application only. Please visit Business School website for more information<br/><br/>",
+        "processed": "65WAM || 72UOC || room in degree for this course || good academic standing || COMM6000 Career Accelerator: Essentials || COMM1999 First Year Portfolio. This course is by application only. Please visit Business School website for more information"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": "(COMM6000 || COMM1999) && 65WAM && 72UOC",
+        "warning": "This course is by application only. Please visit Business School website for more information."
+    }
+
+def COMM_3030():
+    """
+        "original": "\u2022\tminimum WAM of 65;<br/>\u2022\thave completed a minimum of 48 UOC at the commencement of this course;<br/>\u2022\thave room in their degree for this course;<br/>\u2022\tare in good academic standing;<br/>\u2022\thave completed COMM6000 Career Accelerator: Essentials or COMM1999 First Year Portfolio (Business School students only)<br/>This course may count as a Business School Free Elective, OR as a major elective within some schools. It is recommended a progression check is completed prior to enrolling.<br/><br/>",
+        "processed": "\u2022\t65WAM; \u2022\thave a minimum 48UOC at the commencement of this course; \u2022\thave room in their degree for this course; \u2022\tare in good academic standing; \u2022\thave COMM6000 Career Accelerator: Essentials || COMM1999 First Year Portfolio (Business School only) This course may count as a Business School Free Elective || as a major elective within some schools. It is recommended a progression check is prior to enrolling"
+    """
+
+    return "(COMM6000 || COMM1999) && 65WAM && 72UOC"
+
+def COMM_3090(conditions):
+    """
+        "original": "Students are expected to be in their final year of a Bachelor of Commerce single or dual degree with at least 108 UOC completed.<br/><br/>",
+        "processed": "are expected to be in their final year of a Bachelor of Commerce single || dual degree with 108UOC"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": "108UOC",
+        "warning": "Students are expected to be in their final year of a Bachelor of Commerce single or dual degree with at least 108 UOC completed."
```

Comment:
Shorten warning to "Students are expected to be in their final year of a Bachelor of Commerce single or dual degree"